test/integration/podgc/podgc_test.go

```diff
@@ -172,112 +175,119 @@ func TestTerminatingOnOutOfServiceNode(t *testing.T) {
 			enablePodDisruptionConditions: false,
 			withFinalizer:                 false,
 		},
+		"pod is getting deleted when a finalizer and PodDisruptionConditions disabled": {
+			enablePodDisruptionConditions: false,
+			withFinalizer:                 true,
+		},
 	}
 
 	for name, test := range tests {
-		t.Run(name, func(t *testing.T) {
-			defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.PodDisruptionConditions, test.enablePodDisruptionConditions)()
-			defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.NodeOutOfServiceVolumeDetach, true)()
-			testCtx := setup(t, "podgc-out-of-service")
-			cs := testCtx.ClientSet
+		for _, podReplacementPolicyEnable := range []bool{false, true} {
```

Comment:
Similarly here, please declare `enablePodReplacementPolicy` it as a testcase parameter.

Reply:
done.